lib/cryptoexchange/exchanges/bitstamp/services/pairs.rb

```diff
@@ -0,0 +1,26 @@
+module Cryptoexchange::Exchanges
+  module Bitstamp
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        MARKET = Bitstamp::Market
+
+        def fetch
+          output = super
+          adapt(output)
+        end
+
+        def adapt(output)
+          market_pairs = []
+          output.each do |pair|
+            market_pairs << Bitstamp::Models::MarketPair.new(
```

Comment:
@Swingcloud appreciate if you can nest the args righ around `Bitstamp::Models::MarketPair`, thanks